src/test/java/com/iexec/core/metric/MetricServiceTests.java

```diff
@@ -78,4 +91,46 @@ void shouldGetPlatformMetrics() {
         assertThat(metric.getLatestBlockNumberWithDeal()).isEqualTo(255);
     }
 
+    @Test
+    void shouldInitializeMetrics() {
+        verifyGaugeMetric("iexec.core.workers", 3);
+        verifyGaugeMetric("iexec.core.cpu.total", 1);
+        verifyGaugeMetric("iexec.core.cpu.available", 1);
+        verifyGaugeMetric("iexec.core.tasks.completed", 0);
+        verifyGaugeMetric("iexec.core.deals.events", 10L);
+        verifyGaugeMetric("iexec.core.deals", 8);
+        verifyGaugeMetric("iexec.core.deals.replay", 2L);
+        verifyGaugeMetric("iexec.core.deals.last.block", 255L);
```

Comment:
These magic numbers could be converted to constants. It may make those tests more easy to maintain and update later on.

Reply:
public static final String METRIC_WORKERS = "iexec.core.workers";
    public static final String METRIC_CPU_TOTAL = "iexec.core.cpu.total";
    public static final String METRIC_CPU_AVAILABLE = "iexec.core.cpu.available";
    public static final String METRIC_TASKS_COMPLETED = "iexec.core.tasks.completed";
    public static final String METRIC_DEALS_EVENTS = "iexec.core.deals.events";
    public static final String METRIC_DEALS_COUNT = "iexec.core.deals";
    public static final String METRIC_DEALS_REPLAY = "iexec.core.deals.replay";
    public static final String METRIC_DEALS_LAST_BLOCK = "iexec.core.deals.last.block";

Reply:
I was also talking about the values (3, 1, 1, 0, 10L, 8, 2L, 255L) :)